src/Entities/Entity.cpp

```diff
@@ -222,8 +222,15 @@ void cEntity::Destroy(bool a_ShouldBroadcast)
 	}
 
 	cChunk * ParentChunk = GetParentChunk();
-	m_World->QueueTask([this, ParentChunk](cWorld & a_World)
+
+	// We remove the entity from memory 5 ticks later, giving everyone plenty of time
```

Comment:
This sounds off. Are you talking about threads? Why is it not possible for "someone" to actually need > 5 seconds to read IsTicking?